React-tutorial/my-app/src/Board.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+import { Square } from './Square';
+import { calculateWinner } from './calculatteWinner';
+
+interface BoardProps {
+  // squares: Array<string>;
+}
+
+interface BoardState {
+  squares: Array<string>;
+  xIsNext: boolean;
+}
+
+export class Board extends React.Component<BoardProps, BoardState> {
+  constructor(props: BoardProps) {
+    super(props);
+    this.state = {
+      squares: Array(9).fill(''),
+      xIsNext: true,
+    };
+  }
+
+  handleClick(i: number) {
```

Comment:
```suggestion
  private handleClick(i: number) {
```
TypeScriptだとデフォルトでpublicとのことです
http://js.studio-kingdom.com/typescript/handbook/classes